apps/files/src/components/BreadCrumbs.vue

```diff
@@ -86,19 +110,19 @@ export default Vue.extend({
 	},
 
 	methods: {
-		getNodeFromId(id) {
+		getNodeFromId(id: number): Node | undefined {
 			return this.filesStore.getNode(id)
 		},
-		getFileIdFromPath(path) {
+		getFileIdFromPath(path: string): number | undefined {
 			return this.pathsStore.getPath(this.currentView?.id, path)
 		},
-		getDirDisplayName(path) {
+		getDirDisplayName(path: string): string | TypeError {
```

Comment:
```suggestion
		getDirDisplayName(path: string): string {
```

There is no TypeError returned but only strings. TypeError might be thrown.